lib/msal-angular/README.md

```diff
@@ -148,4 +153,4 @@ Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT
 
 ## We Value and Adhere to the Microsoft Open Source Code of Conduct
 
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
```

Comment:
What was the diff here?

Reply:
Not sure why this came up with a diff. Nothing was changed here.